src/OpenLoco/Objects/VehicleObject.cpp

```diff
@@ -361,4 +366,27 @@ namespace OpenLoco
 
         std::fill(std::begin(startSounds), std::end(startSounds), 0);
     }
+
+    // 0x004B9780
+    uint32_t VehicleObject::getLength() const
+    {
+        auto length = 0;
+        for (auto i = 0; i < var_04; ++i)
+        {
+            if (var_24[i].body_sprite_ind == 0xFF)
+            {
+                continue;
+            }
+
+            auto unk = var_24[i].body_sprite_ind & (VehicleObject::kMaxBodySprites - 1);
+            length += bodySprites[unk].bogey_position * 2;
+        }
+        return length;
+    }
+
+    float VehicleObject::getDisplayLength() const
```

Comment:
No floats. I would prefer it if you just created a function in the StringManager something like `internalLengthToComma1DP`
```cpp
int32_t internalLengthToComma1DP(int32_t length) {
    return length * 100 / 4 / 32;
}
```